test/e2e/instrumentation/events.go

```diff
@@ -147,6 +147,24 @@ var _ = common.SIGDescribe("Events API", func() {
 			framework.Failf("test event wasn't properly patched: %v", diff.ObjectReflectDiff(testEvent, event))
 		}
 
+		ginkgo.By("updating the test event")
```

Comment:
This addition looks fine. Please just extend the test comment in line 91 to reflect this extension.

Reply:
Done.